### PR TITLE
Feat/places ingestion

### PR DESCRIPTION
**Pull request by Sabin-Subedi**

### CODE REVIEW

File: src/server/scripts/fsq_ingestion.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+            Authorization: env.foursquare.apiKey,
+          },
+        })
+      : await axios.get(FOURSQUARE_BASE_URL, {
+          headers: {
+            Authorization: env.foursquare.apiKey,
+          },
+          params: {
+            ll: `${lat},${lon}`,
+            radius: 3000,
```
Comment (Ashmit101) [inline src/server/scripts/fsq_ingestion.ts:78]: Will this radius give enough results?

**Review by Ashmit101 — DISMISSED** [review]

File: src/server/scripts/fsq_ingestion.ts (hunk 1 above)
Comment (Sabin-Subedi) [reply src/server/scripts/fsq_ingestion.ts:78]: actually we have to change radius and ingest to get more results which is already ingested

**Comment by lucerowb** [comment]
need rebasing

**Comment by lucerowb** [comment]
Still build is failing
CleanShot 2024-12-24 at 5  00 55@2x

**Review by lucerowb — APPROVED** [review]

**Review by lucerowb — APPROVED** [review]

**Pull request merged**